exchanges/deribit/deribit_websocket.go

```diff
@@ -92,6 +92,10 @@ var (
 			"interval": 15,
 		},
 	}
+
+	// loadedOBPairMap to keep track of instruments with their orderbook snapshot loaded
+	// we use this to check whether we need to load a snapshot or update if snapshot is already loaded
+	loadedOBPairMap = map[string]bool{}
```

Comment:
This doesn't look correct:

1) If the bot disconnects and reconnects or we disable or enable a pair then re-enable it later this will be set to true already. 
2) If the label is snapshot then it should load as a snapshot else it should be a change update. So I don't know if this is required. I don't know why something being labelled not a snapshot be loaded differently.